libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -276,6 +281,35 @@ public void setAztecVideoLoader(Html.VideoThumbnailGetter videoLoader) {
         this.aztecVideoLoader = videoLoader;
     }
 
+    private InputFilter replaceEnterKeyWithSpaceInputFilter = new InputFilter() {
+        @Override
+        public CharSequence filter(CharSequence source, int start, int end,
+                                   Spanned dest, int dstart, int dend) {
+            //  You sometimes get a SpannableStringBuilder, sometimes a plain String in the source parameter
+            if (source instanceof SpannableStringBuilder) {
```

Comment:
I've read the comment, but I wonder if we're adding spans to the title and that's why we sometimes have a plain string or a spannable. I don't think any span is needed for the title so maybe that exhibits an underlying issue.

I also wonder if we should use a simple EditText for the title as we don't need Aztec features there, cc @0nko maybe I'm missing something.

Reply:
The Android doc says it could be a SpannableStringBuilder or a simple string, so i've added both cases to be sure. 

If we decide to drop Aztec in favour of EditText, the code that replaces `Enter` with `Space` is also required, since we will have the same issue on longer titles. This PR should not change much, I guess at all.

Reply:
Yep I read the doc, but we get a `SpannableStringBuilder` because there is a span in that source string. I don't think there should be one in there, that's why I'm a bit worried.

Reply:
I've replaced AztecText with EditText and once `Enter key` is entered in the title we get `SpannableStringBuilder`. I guess it's the Android FW that adds it.

Reply:
Argh, ok that's weird.

Reply:
Good call on changing it to EditText 👍